sites/mosul/configs/openmrs/initializer_config/ampathformstranslations/F41-ER_triage_form_translations_ar.json

```diff
@@ -0,0 +1,171 @@
+{
+  "uuid": "243661ad-0ea3-42fe-a5ed-f9dc259ee118",
+  "form": "F41-ER triage form",
+  "description": "Ar Translations for 'F41-ER triage form'",
+  "language": "ar",
+  "translations": {
+    "1 - 6 months": null,
+    "1 hour or less": null,
+    "1 to 4 years": null,
+    "120 hours - 1 month": null,
+    "13-24 hours": null,
+    "2-6 hours": null,
+    "24 hours or more": null,
+    "5 to 12 years": null,
+    "6 months - 1 year": null,
```

Comment:
Is this still a WIP? The translation strings are missing

Reply:
No, translations are added!